src/_locales/es-ES.json

```diff
@@ -0,0 +1,96 @@
+{
+  "audio-device-communications": "Sistema de comunicaciones por defecto",
+  "audio-device-default": "Dispositivo por defecto",
+  "audio-device-unknown": "Dispositivo Desconocido",
+
+  "button-test-ok": "Aceptar",
+  "button-text-close": "Cerrar",
+  "button-text-dont-tell-me-again": "No recordar más",
+  "button-text-lets-do-this": "Hagámoslo!!",
+  "button-text-lets-go": "Vamos!!",
+  "button-text-not-now": "Ahora mismo no",
+
+  "label-about": "Acerca de",
+  "label-desktop-settings": "Ajustes de escritorio",
+  "label-donate": "Donar",
+  "label-help": "Ayuda",
+  "label-issues": "Problemas",
+  "label-learn-more": "Aprender más",
+  "label-loading-devices": "Cargando Dispositivos...",
+  "label-pause-after-song": "Pausar después de esta canción",
+  "label-quit": "Salir",
+  "label-show-lyrics": "Mostrar Letras (Beta)",
+  "label-unknown-artist": "Artista desconocido",
+  "label-unknown-song": "Canción desconocida ",
```

Comment:
"canción" implies a musical genere that always include lyrics, please consider using "pista" which is a more generic name for a musical piece.